kitty/pass_keys.py

```diff
@@ -89,18 +86,21 @@ def handle_result(args, result, target_window_id, boss):
     window = boss.window_id_map.get(target_window_id)
     action = args[1]
     direction = args[2]
-    key_mapping = args[3] if action == 'neighboring_window' else args[4]
-    amount = int(args[3]) if action == 'relative_resize' else None
-    vim_id_idx = 4 if action == 'neighboring_window' else 5
-    vim_id = args[vim_id_idx] if len(args) > vim_id_idx else "n?vim"
+    key_mapping = args[3] if action == "neighboring_window" else args[4]
+    amount = int(args[3]) if action == "relative_resize" else None
 
     if window is None:
         return
-    if is_window_vim(window, vim_id):
+
+    cmd = window.child.foreground_cmdline[0]
+
+    # keyboard_mode_name(window.screen) == "kitty"  You can get kitty.
+    # result: zsh, tmux, nvim
+    if cmd[-4:] == "nvim":
```

Comment:
correct me if I'm wrong but I don't think this works if, for example, you run `nvim ./some/file/name.txt` or if you run Neovim via a `vim` shell alias.